docs/specifications/tests/DNSQueryAndResponseDefaults.md

```diff
@@ -0,0 +1,163 @@
+# DNS Query and Response Defaults
+
+**Table of contents**
+* [Overview](#Overview)
+* [Application of this specification](#application-of-this-specification)
+* [Default setting in *DNS Query*](#default-setting-in-dns-query)
+* [Default setting in *EDNS Query*](#default-setting-in-edns-query)
+* [Default setting in *DNSSEC Query*](#default-setting-in-dnssec-query)
+* [Default handling of a *DNS Response*](#default-handling-of-a-dns-response)
+* [Default handling of an *EDNS Response*](#default-handling-of-an-edns-response)
+* [Default handling of a *DNSSEC response*](#default-handling-of-a-dnssec-response)
+
+
+## Overview
+
+Almost all [test cases] emit DNS queries and react on the responses. This
+document defines the default "setting" of a DNS query and the default handling
+of "setting" in the DNS response. The meaning of "setting" here will be clear
+from the specification below.
+
+
+## Application of this specification
+
+This specification applies to all [test case specifications][Test Cases] that has
+an explicit reference to this document.
+
+
+## Default setting in *DNS Query*
+
+A *DNS Query* has the following default setting. A test case specification can
+refer to a *DNS Query* with one or several changes to the *Setting Items*
+overriding the default values. If a *Setting Item* is specified as "fixed" then
+the default value cannot be overidden.
+
+|Setting Item |Default value |Fixed |Comment                       |
+|:------------|:-------------|:-----|:-----------------------------|
+|Protocol     | UDP          |      |                              |
+|OpCode       | "query"      | yes  |                              |
+|QR flag      | unset        | yes  | Always unset in a query      |
+|AA flag      | unset        | yes  |                              |
+|TC flag      | unset        | yes  |                              |
+|RD flag      | unset        |      |                              |
+|RA flag      | unset        | yes  |                              |
+|AD flag      | unset        |      |                              |
+|CD flag      | unset        |      |                              |
+|RCODE        | "NoError"    | yes  |                              |
+|Query name   | -            |      | Must be defined in test case |
+|Query type   | -            |      | Must be defined in test case |
+|Query class  | "IN"         | yes  |                              |
```

Comment:
Perhaps that "class"  should not declare as a "fixed" value. Although we use "IN" in the current TC specified, I think we should not "fix" it for future usage.

Reply:
I really do not see the use case of other class than "IN", but I have updated the specification to make "IN" the default instead of fixed. Please re-review.